zimui/src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { defineCustomElement } from 'vue'
+import Navbar from './components/DevdocsNavbar.vue'
+
+// Convert the navbar into a custom element. We're using this rather than
+// the shorthand .ce.vue naming because we need to disable shadowRoot
+// so the DevDocs CSS will work on the navigation entries.
+const DevdocsNavbar = defineCustomElement(Navbar, { shadowRoot: false })
```

Comment:
This feels really hackish. Don't we have other possibilities? In Vue.JS boilerplate we have a `style.css` which is imported in `main.ts` and available to the whole application, shouldn't we simply do the same with `application.css`?

Reply:
We currently fetch that dynamically because it's compiled with language specific overrides and I wanted a build of the scraper to work long into the future without being tied to a specific DevDocs release. it seemed like there would be less maintenance that way.

Similarly, it seemed like if we forked the navbar style it would eventually start coming out of sync with DevDocs proper (which we'll still need to pull if we want language doc style overrides). so using cascading styles in our custom element seemed like a cleaner approach that wouldn't leave us chasing styles.

I'm happy to swap to using a cached version or a re-implementation if you'd prefer.